test/acceptance/todo-cli-test.js

```diff
@@ -340,6 +340,75 @@ describe('todo usage', () => {
       expect(todoDirs).toHaveLength(0);
     });
 
+    it('errors if a todo item has expired when running without params', async function () {
+      project.setConfig({
+        rules: {
+          'require-button-type': true,
+        },
+      });
+
+      project.write({
+        app: {
+          templates: {
+            'require-button-type.hbs': '<button>Check Expiration</button>',
+          },
+        },
+      });
+
+      project.writeTodoConfig({
+        error: 5,
+      });
+
+      // generate todo based on existing error
+      await run(['.', '--update-todo'], {
+        // change the date so errorDate is before today
+        env: {
+          TODO_CREATED_DATE: subDays(new Date(), 10).toJSON(),
+        },
+      });
+
+      // run normally and expect an error for not running --clean-todo
+      let result = await run(['.']);
+
+      expect(result.exitCode).toEqual(1);
+      expect(result.stdout).toMatchInlineSnapshot(`
+        "app/templates/require-button-type.hbs
+          1:0  error  All \`<button>\` elements should have a valid \`type\` attribute  require-button-type
+          -:-  error  Expired todos exist. Please run \`ember-template-lint --clean-todo\` to remove expired todos.  expired-todo-rule
+
+        ✖ 2 problems (2 errors, 0 warnings)
+          2 errors and 0 warnings potentially fixable with the \`--fix\` option."
+      `);
+
+      // run clean-todo, and expect that this will delete the expired todo item
+      await run(['app/templates/require-button-type.hbs', '--clean-todo']);
```

Comment:
This can be removed, since we're _always_ removing expired.

Reply:
I think this removes the only place where the flag was tested. We okay with that?

Reply:
But the invocation on line 371 removes expired todos, right? We're always removing expired todos, and you've explicitly created a todo that's expired, right? So, after line 371, this todo that we want to check for no longer exists, right? 

Why do we need to run `--clean-todo`?